src/threads/infrastructure/persistence/relational/entities/thread.entity.ts

```diff
@@ -0,0 +1,23 @@
+import { Entity, ManyToOne, PrimaryGeneratedColumn } from 'typeorm';
+import { Thread } from '../../../../domain/thread';
+import { EntityRelationalHelper } from '../../../../../utils/relational-entity-helper';
+import { UserEntity } from '../../../../../users/infrastructure/persistence/relational/entities/user.entity';
+import { MessageEntity } from '../../../../../messages/infrastructure/presistence/entities/message.entity';
+
+@Entity({
+  name: 'thread',
+})
+export class ThreadEntity extends EntityRelationalHelper implements Thread {
```

Comment:
The entity is used to track who participates in a thread, so we should name it as ThreadParticipants